src/oci-umount.c

```diff
@@ -369,21 +432,34 @@ static int prestart(const char *rootfs,
 	}
 
 	while ((read = getline(&line, &len, fp)) != -1) {
+		bool submounts_only = false;
```

Comment:
Comment handling and rewritten oci-umount.conf file in this pr https://github.com/projectatomic/oci-umount/pull/14